pyproject.toml

```diff
@@ -29,6 +29,7 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
+    "jupyter-client>=8.6.3",
```

Comment:
issue: I believe we should make it optional (will need update to the installation/usage docs), as users who are not interested in TypeScript will not want to pull in jupyter-client and all its dependencies. This might change in the future if we use Jupyter kernels for all languages but that's not the case for now :slightly_smiling_face: